Gopkg.lock

```diff
@@ -189,6 +189,15 @@
   revision = "c01d1270ff3e442a8a57cddc1c92dc1138598194"
   version = "v1.2.0"
 
+[[projects]]
```

Comment:
Have we determined if this will cause a dependency cycle since go-pilosa depends on pilosa?